pipelines/datasets/br_bd_diretorios_brasil/utils.py

```
@@ -0,0 +1,438 @@
+            "Check that the INEP portal is reachable and the catalog path is still valid."
+        )
+
+    size_mb = csv_path.stat().st_size / 1_048_576
+    log.info("Downloaded %.1f MB → %s", size_mb, csv_path)
+    return csv_path
+
+
+def _curl(
+    cmd: list[str], *, label: str, capture_stdout: bool = False
+) -> subprocess.CompletedProcess:
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)